CONTRIBUTING.md

```diff
@@ -0,0 +1,167 @@
+# ROCM-CORE Contributing Guide
+To ensure the quality of the ROCM-CORE code base, the ROCM-CORE team has 
+established a code review process to inform developers of the steps 
+that are required to shepherd a change-set into the repository.
+
+#### Table Of Contents
+
+[How to get started](#How-to-get-started)
+
+[How do I contribute?](#how-do-i-contribute)
+  * [Reporting Issues](#reporting-issues)
+  * [Creating a Pull Request](#Creating-a-Pull-Request)
+
+[Responsibility of the Author](#Responsibility-of-the-Author)
+
+[Responsibility of the Reviewer](#Responsibility-of-the-Reviewer)
+
+[The Review](#the-review)
+
+[References](#References)
+## How to get started
+rocm-core is a utility which can be used to get ROCm release version. 
+It also provides the Lmod modules files for the ROCm release.
+getROCmVersion function provides the ROCm version. 
+
+## How do I contribute
+### Deliverables
+All contributions you make will be under the [MIT Software License](copyright).
+For each new file in repository, 
+Please include the licensing header
+```
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 20xx Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+```
+
+### Formatting the code
+
+All the code is formatted using `clang-format`. To format a file, use:
+
+```shell
+clang-format-10 -style=file -i <path-to-source-file>
+```
+
+To format the code per commit, you can install githooks:
+
+```shell
+./.githooks/install
```

Comment:
Do we have this .githooks/install file?

Reply:
Current version we dont have this install update, so removing this githooks install operations in the latest version. Will check this and update in upcoming versions once this scripts are available